Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -25,6 +25,9 @@
 except Exception:
     pass
 
+# when returning an entry using `demisto.results` or `return_outputs` that has timeline data,
+# use the TIMELINE_CATEGORY constant as the value for the "Category" field of the timeline dictionary
+TIMELINE_CATEGORY = 'Integration Update'
```

Comment:
if this is always the case, why are we not setting it ourselves in `demisto.results` ?

Reply:
Because there are, in theory, cases where you wouldn't use this category value. For instance, let's say you're doing stuff (calling `demisto.results()` or `return_outputs`) from a script - the category would probably be different (could be `"Automation Update"` maybe? Who knows?). Plus, I imagine that new categories will be added in the future.

Reply:
if there will be such case we will handle it, but for now i want to ease the process of development
if the only possible value for the field is currently, that is demisto 5.5, `Integration Update` then lets set it in `results`

Reply:
tov 👍